go-controller/pkg/node/port_claim.go

```diff
@@ -7,7 +7,6 @@ import (
 	"sync"
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
-
```

Comment:
keep this space? it helps to see what we import externally vs. from within the project